modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form-report/components/list/List.js

```diff
@@ -28,13 +30,27 @@ export default function List({data, field, summary, totalEntries, type}) {
 
 		return moment(field).locale(locale).format('L');
 	};
+	const formatDateTime = (field) => {
+		const locale = themeDisplay.getLanguageId().split('_', 1).join('');
+
+		const date = moment(field).locale(locale).format('L');
+		const time = moment(field).locale(locale).format('LT');
+
+		const [hourFormat] = time.split(NOT_DIGIT_REGEX, 1);
+
+		const formattedTime = hourFormat.length === 1 ? `0${time}` : time;
+
+		return `${date} ${formattedTime}`;
+	};
 
 	const checkType = (field, type) => {
 		switch (type) {
 			case 'color':
 				return <Color hexColor={field} />;
 			case 'date':
 				return formatDate(field);
+			case 'date_time':
+				return formatDateTime(field);
```

Comment:
According to the change I've requested before:
```suggestion
				return formatDate(field, true);
```